crates/cac_demo_app/README.md

```diff
@@ -0,0 +1,21 @@
+# Superposition Demo App setup instructions
+
+1. Clone the repo
+```sh
+git clone https://github.com/knutties/superposition-demo-app.git
+```
+
+2. Import the database using 
+```sh
+psql --username=postgres config < data/demo_app_db_dump.sql
+```
+
+3. Compile and run the app
+```sh
+cargo run --bin cac-demo-app
```

Comment:
```suggestion
cd crates/cac_demo_app
cargo run --bin cac-demo-app
```

Reply:
You don't need to cd to the directory, the project is part of the workspace